baybe/searchspace/continuous.py

```diff
@@ -42,6 +44,36 @@ class SubspaceContinuous(SerialMixin):
     )
     """List of linear inequality constraints."""
 
+    def __repr__(self) -> str:
+        """Override the standard __repr__."""
+        if self.is_empty:
+            return ""
+        # Convert the lists to dataFrames to be able to use pretty_printing
+        par_df = parameter_cartesian_prod_to_df(self.parameters)
+        const_lin_eq_df = parameter_cartesian_prod_to_df(self.constraints_lin_eq)
+        const_lin_ineq_df = parameter_cartesian_prod_to_df(self.constraints_lin_ineq)
+
+        # Put all attributes of the continuous class in one string
+        continuous_str = (
```

Comment:
This "string addition" approach you use here is quite old school style. Can we use nice f-strings instead?

Reply:
Thanks! I refactored the strings to f-strings 👍